packages/styles/forms.css

```diff
@@ -463,3 +463,8 @@ textarea.Field--has-error:focus:hover,
 .Checkbox__overlay:hover:not(.Checkbox__overlay--disabled) {
   border: 1px solid currentColor;
 }
+
+.Checkbox__overlay--disabled.Icon--checkbox-checked,
+.Checkbox__overlay--disabled.Icon--checkbox-unchecked {
```

Comment:
I'm wondering if this should be added in the above rules (lines 439~447) so it covers both checkbox and radio instead of an additional rule?

Reply:
@scurker Done.